axelarnetwork/axelar-core
# feat(axelarnet): mint/burn cosmos based asset on evm

## Description
resolves #693 
- mint and burn Axelarnet native asset on EVM chains
- mint and burn Cosmos based assets (through ibc) on Axelar chains to EVM chains 
- send minted Satoshi on Axelar network to EVM chains. 
 
## Todos


## Steps to Test
Assume there is a ibc connection establish between two chians
1. register a chain
```
axelard tx axelarnet add-cosmos-based-chain [chain] [native asset]
```
2. register a tracing path for an asset 
```
axelard tx axelarnet register-path [native asset] "transfer/channel-0" 
```
3. link to evm
```
axelard tx axelarnet link ethereum 0x68B93045fe7D8794a7cAF327e7f855CD6Cd03BB8 [native asset]
```
4. send the native asset to linked address the token is represented by ibc/{hash} e.g.
```
axelard tx bank send [from] [to] 2000"ibc/C053D637CCA2A2BA030E2C5EE1B28A16F71CCB0E45E8BE52766DC1B241B77878" 
```
5. confirm the deposit e.g.
```
axelard tx axelarnet confirm-deposit [txHash]  2000"ibc/C053D637CCA2A2BA030E2C5EE1B28A16F71CCB0E45E8BE52766DC1B241B77878" [linked addr]
```
## Expected Behaviour
We enforce one ibc transferred asset can only have one registered tracing path. We will discuss how to improve later.
## Other Notes
